<launch>
	
	<include file="$(find techxrobot)/launch/test_obs_avoid.launch"/>
        <include file="$(find directed_exploration)/launch/de_planner.launch"/>
  
		<node pkg="test_nav" type="test_nav" name="test_nav" output="log">
			  <param name="initial_bearing" value="2.38411976"/>
			  <param name="waypoint_fname" value="/home/techx/ros_workspace/test_nav/launch/waypoints_test_1.txt"/>			  
			  <param name="gps_topic" value="/drrobot_gps_info"/>
			  <param name="initial_latitude" value="1.342717"/>
			  <param name="initial_longitude" value="103.680319"/>
			  <param name="gps_timeout_th" value="15.0"/>
			  <param name="gps_avg_N" value="10"/>
			  <param name="compass_topic" value="/drrobot_compass_info"/>
			  <param name="use_ogmapper" value="false"/>
			  <param name="inter_beacon_dist" value="6.0"/>
			  <param name="min_dist_from_beacon_to_wp" value="6.0"/>
			  <param name="neighborhood_timeout_threshold" value="20"/>
			  <param name="neighborhood_radius" value="5.0"/>
			  <param name="target_epsilon" value="1.0"/>			  
		</node>
</launch>
